cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -514,3 +513,33 @@ def test_rebuilt_vcenter_duplicate_hosts(appliance, provider):
     # Using appliance.rest_api as hosts.all() do not return archived hosts, I need those too
     hosts_after = len(appliance.rest_api.collections.hosts.all)
     assert hosts_before == hosts_after
+
+
+@pytest.mark.tier(1)
+@pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE, override=True)
+@pytest.mark.meta(automates=[1755070])
+def test_vm_notes_ui(appliance, provider, soft_assert):
+    """
+    Check if the VM Notes are shown in the CFME UI for VMs on VMware.
+
+    Bugzilla:
+        1755070
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/2h
+        testtype: functional
+        testSteps:
+            1.Add VMware provider to CFME
+            2.Navigate to Compute->Infrastructure->Virtual Machines
+            3.Select a VM to view details and click on "Container" in the "Basic Information" table
+            4.Check if Notes field exists and Notes are shown
+    """
+    vms_collections = appliance.collections.infra_vms
+    vm = vms_collections.instantiate(name='cu-24x7', provider=provider)
+    view = navigate_to(vm, 'VmContainer')
+    assert view.basic_information.get_field('Notes').is_displayed
+    # all cu-24x7 do have notes on them hence following check would work(but not required)
+    soft_assert(view.basic_information.read()['Notes'] != '')
```

Comment:
Since this is the last assert, why use `soft_assert` here?

Reply:
I am just checking if there are notes, all cu24x7 vms do have notes, but in case there are none, I don't want to make it a hard failure, hence soft_assert. Its valid to have empty notes.

Reply:
soft_asserts are still test failures, they just don't stop test execution. Gotta make a decision on whether you want to consider this a failure, or just log the fact. What's the benefit?